packages/radix-vue/src/Combobox/ComboboxRoot.vue

```diff
@@ -256,6 +265,9 @@ provideComboboxRootContext({
       selectedValue.value = filteredOptions.value[val === 'up' ? index - 1 : index + 1]
 
     scrollSelectedValueIntoView()
+    focusOnSelectedElement()
+
+    nextTick(() => inputElement.value?.focus({ preventScroll: true }))
```

Comment:
similarly this is not required

Reply:
> similarly this is not required

Same here - focus announces the option, then this returns the focus back the input field - just as how one would expect a combobox to work.